docs/src/config-file.md

```diff
@@ -199,6 +199,18 @@ mbtiles:
     # named source matching source name to a single file
     mb-src1: /path/to/mbtiles1.mbtiles
 
+# Cloud Optimized GeoTIFF File Sources
+cog:
+  paths:
+    # scan this whole dir, matching all *.tif files
+    - /dir-path
+    # specific tif file will be published as a cog source
+    - /path/to/pmt.pmtiles
```

Comment:
```suggestion
    # specific TIFF file will be published as a cog source
    - /path/to/cogfile.tif
```

Reply:
@nyurik  Yes they are seperated.
* [tiff 6.0 doc](extension://bfdogplmndidlpjfhoijckpakkdjkkil/pdf/viewer.html?file=https%3A%2F%2Fwww.itu.int%2Fitudoc%2Fitu-t%2Fcom16%2Ftiff-fx%2Fdocs%2Ftiff6.pdf)
* [geotiff doc](https://docs.ogc.org/is/19-008r4/19-008r4.html)
* [cog doc](https://docs.ogc.org/is/21-026/21-026.html)